arangod/RestHandler/RestVocbaseBaseHandler.cpp

```diff
@@ -683,7 +683,10 @@ RestVocbaseBaseHandler::createTransactionContext(AccessMode::Type mode) const {
           "illegal to start a managed transaction here");
     }
     if (value.compare(pos, std::string::npos, " aql") == 0) {
-      return std::make_shared<transaction::AQLStandaloneContext>(_vocbase, tid);
+      auto aqlStandaloneContext =
+          std::make_shared<transaction::AQLStandaloneContext>(_vocbase, tid);
+      aqlStandaloneContext->setStreaming();
```

Comment:
it's because, if you have reached this region, it means that the transaction id was in the header of the request, as it happens after the 
```
 if (!found) {
    return std::make_shared<transaction::StandaloneContext>(_vocbase);
  }
 ``` 
 in line 659